myuw/static/js/card/schedule/visual.js

```diff
@@ -167,7 +183,28 @@ var VisualScheduleCard = {
                         }
                     }
                 });
+
+                if(section.has_eos_dates) {
+                    // sort the meeting by eos_start_date
+                    section.meetings.sort(function(m1, m2) {
```

Comment:
Could this function be refactored out of here and into a file level function or put in a utility file?